client/src/actions/index.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+export const CREATE_MEAL = 'CREATE_MEAL';
+
+const ROOT_URL = 'http://localhost:3001';
+export function createMeal(values, cb) {
+  const request = axios.post(`${ROOT_URL}/api/chefs/meals`, values)
```

Comment:
The request to the server can be made without with the `ROOT_URL`. That is, request can be changed to 
```javascript
const request = axios.post(/api/chefs/meals, values)
```